meson.build

```diff
@@ -508,3 +508,31 @@ if get_option('enable-test')
 
   subdir('tests')
 endif
+
+# Generates nnstreamer documentation using hotdoc
+if get_option('enable-doc')
+  hotdoc_p = find_program('hotdoc')
+  if not hotdoc_p.found()
+      message('Hotdoc not found, not building the documentation')
+      subdir_done()
+  endif
+
+  hotdoc_req = '>= 0.12.2'
+  hotdoc_version = run_command(hotdoc_p, '--version').stdout()
+  if not hotdoc_version.version_compare(hotdoc_req)
+      error('Hotdoc version @0@ not found, got @1@'.format(hotdoc_req, hotdoc_version))
+  endif
+
+  hotdoc = import('hotdoc')
+
+  gstreamer_doc = hotdoc.generate_doc('NNStreamer',
+      project_version: meson.project_version(),
+      sitemap: 'sitemap.txt',
+      index: 'index.md',
+      install: true,
+      syntax_highlighting_activate: true,
+      html_extra_theme: join_paths(meson.current_source_dir(), 'theme/extra'),
+      disable_incremental_build: true,
+      build_always_stale: true,
+  )
+endif
```

Comment:
How about moving this and related files into `./Documentation`? e.g., `Documentation/meson.build`

Reply:
I agree with some of your opinions.
However, The source code such as `gst` and `ext` will be parsed and the API documents will be generated later.
So, I think it's better to put it here.

Reply:
Since API reference is just linked doxygen book, I moved meson command as you suggested. Thanks.